docker-compose.yml

```diff
@@ -2,9 +2,7 @@ version: "3.9"
 
 services:
   presto:
-    image: ahanaio/prestodb-sandbox:0.281
-    # there is no arm64 version of this image
-    platform: linux/amd64
```

Comment:
Can remove the `platform` flag here as our fork publishes both `linux/amd64` and `linux/arm64` for each version tag.

Reply:
Turns out `linux/amd64` is still required. Not sure exactly what configuration I had tested to believe it worked on `linux/arm64`, :shrug:

Reply:
Alright, looks like there were a few bugs, and that having squashed them, it's now that `popsql/prestodb-sandbox:0.281` only supports `linux/amd64` but that `0.282+` supports that and `linux/arm64`. I'm really not sure what my local state was last week, but retested everything from scratch and it looks good. :+1: